app/preview/views.py

```diff
@@ -0,0 +1,46 @@
+import os
+import tempfile
+
+from flask import Blueprint, request, jsonify
+
+from flask.helpers import make_response, send_from_directory
+from app.doc2html.views import process_html_path
+
+htmlByBLobBlueprint = Blueprint('doc2htmlByBLob', __name__)
+
+
+@htmlByBLobBlueprint.route("/doc2htmlByBLob", methods=["POST"])
+def doc_2_html_from_bLob():
+    file = request.files.get('doc')
+    resp = build_response(file)
+    resp.headers.add("Access-Control-Allow-Origin", "*")
+    return resp
+
+
+def build_response(file):
+    if file is None:
+        return make_response("You must send file", 400)
+    with tempfile.TemporaryDirectory() as dir_path:
+        if not os.path.exists(dir_path):
```

Comment:
I think `tempfile.TemporaryDirectory()` creates the directory for you. You don't have to do it yourself

Reply:
removed